tsconfig.base.json

```diff
@@ -15,14 +15,24 @@
     "target": "ESNext",
     "paths": {
       "@expressive-code/core": ["./packages/@expressive-code/core/src"],
+      "@expressive-code/core/hast": ["./packages/@expressive-code/core/src/hast"],
```

Comment:
After adding `core` entry points to the three (3) packages, I started getting typescript warnings/errors due to unresolved packages.  I only needed to add entries here for the new `core` modules but noticed that several others were missing so added those in as well.  If their absence was intentional, let me know and I can remove all but the new `core` ones.